src/Game.py

```diff
@@ -14,16 +14,26 @@ def __init__(self):
 
     def runGame(self):
         pygame.init()
+        clock = pygame.time.Clock()
         pygame.display.set_caption('Basic Pygame program')
         self.isRunning = True
         while self.isRunning:
+            dt = clock.tick(clock.get_fps())/1000
             for event in pygame.event.get():
-                if event.type == QUIT or event.type == KEYDOWN:
+                if event.type == QUIT:
                     self.isRunning = False
+                elif event.type == KEYDOWN:
+                    if event.key == K_ESCAPE:
+                        self.isRunning = False
```

Comment:
I don't see a need to change this from what was previously there.. It is essentially the same and longer.
KEYDOWN stands for Enter

Reply:
KEYDOWN does not seem to work like that..as far as I have seen
if this is the condition -->( if event.type == QUIT or event.type == KEYDOWN: ) pygame quits as soon as I press any key on the keyboard

Reply:
Sounds good